/**:
  ros__parameters:
    enable_capture: True
    capture_interval: 2.0
    sync: False